### DISCUSSION TITLE
Ecosistema Aqara chi puo' far chiarezza fra i piu' esperti sulla compatibilita' allo standar protocollo Mattel ?

### DISCUSSION THREAD

**Discussion by mirrorlessm43** [indomus/forum#3173] [ZigBee, Z-Wave, Thread, Matter]
Un saluto a tutti sono un entusiasta possessore di devices Aqara dal luglio 2023 quando acquistai un controller hub M1s Gen 2  una telecamera G3 e sensoristica varia, fino ad arrivare al distributore spettacolare di croccantini per gatto acquistato recentemente.
Se non ho capito male con i vari aggiornamenti i miei devices e in principal modo il controller M1s dovrebbero essere compatibili anche con il nuovo protocollo standard Mattel , questo che benefici comporta ?
Quali prodotti anche di altre marche sono pienamente compatibili con Aqara per quali utilizzi e funzionalita' ?
Fatemi sapere vi ringrazio saluti Mirrorless M43

**Comment by KC-inDomus** [comment]
Ciao Mirror,
il fatto che l'ecosistema Aqara abbia abbracciato Matter significa i vari "hub" (in realtà sono bridge/gateway) possono essere integrati - asseieme a tutti i componenti zigbee ad essi connessi - ad altri ecosistemi (Alexa, Google Home, ma anche HUB personali come Home Assistant) tramite Matter.

**Reply by mirrorlessm43** [reply]
molto interessante , quindi di fatto si possono integrare in vari eco sistemi di case differenti.
vorrei capire da qualcuno esperto del gruppo come poter valorizzare al meglio i componenti aqara o di altri componenti zigbee all'interno di home assistant , per creare qualcosa di piu' flessibile rispetto al ecostistema aqara basato sul hub aqara m1s gen 2 .
si puo' dedicare un micro pc allo scopo ?
come si configura ?
quali sono le procedure d'installazione e configurazione ?
e' semplice ci sono delle guide / tutorials ?
grazi mirrorless

**Reply by KC-inDomus** [reply]
dove meglio se non qui: https://indomus.it/formazione/primi-passi-con-home-assistant-da-dove-cominciare/
